.github/workflows/create-new-release.yml

```
@@ -0,0 +1,67 @@
+  workflow_dispatch:
+
+permissions:
+  contents: write
+
+jobs:
+  create_release_branch:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)